build_and_install.sh

```diff
@@ -17,8 +17,10 @@ pushd build
       make install DESTDIR=destdir
     popd
   popd
-  cmake -DTd_DIR="$(realpath .)"/td/build/destdir/usr/local/lib/cmake/Td/ -DNoVoip=True ..
-  make -j "${JOBS}"
-  echo "Now calling sudo make install"
-  sudo make install
-popd
+  cmake -DTd_DIR="$(realpath .)"/td/build/destdir/usr/local/lib/cmake/Td/ \
+    -DNoVoip=True \
+    ..
+  popd
+make -j "${JOBS}"
```

Comment:
This `make` command would be run in the original cwd where the `build_and_install.sh` script was called, which is the wrong directory.

Why did this cause errors for you in the first place?

Also, I agree that the directory-navigation here is fishy at best. I suggest inserting a single "anchor" in the very beginning, something like
https://github.com/SerenityOS/serenity/blob/c247aac15efdbeacad6bd41f1d66232a877d01aa/Meta/check-png-sizes.sh#L8-L9